core/lens/lens_column.mli

```diff
@@ -0,0 +1,74 @@
+type t = Types.lens_col
+
+(** Return the name of the column as it would be accessible in links. *)
```

Comment:
Minor comment: What do you mean by "accessible in links"? Also nitpick: s/links/Links.